### DISCUSSION TITLE
Llenar el formulario de Demanda

### DISCUSSION THREAD

**Discussion by jmlimas** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#84] [General]
Yo nunca  he  tenido  una  demanda y menos he demandado  que  datos  voy  a pedir  para  hacer el  formulario de aquì  enviare  el proceso de pago  para  finalizar la  transacción 

<img width="475" alt="Captura de Pantalla 2022-02-22 a la(s) 23 05 39" src="https://user-images.githubusercontent.com/2806970/155264628-dfde360b-71ad-473d-bee5-ee93603f24c2.png">

**Comment by jmlimas** [comment]
<img width="457" alt="Captura de Pantalla 2022-02-22 a la(s) 23 28 36" src="https://user-images.githubusercontent.com/2806970/155266279-0304243c-6728-4a80-847f-9be6485007ff.png">

**Reply by GloriaCoralCerecedo** [reply]
Yo investigue que se necesita para una demanda, lo subí creo aquí creo, son datos del tribunal, del demandaré y demandado, los hechos con pruebas y la petición, más o menos

**Reply by jmlimas** [reply]
Gracias
